src/studio/src/designer/frontend/shared/navigation/drawer/LeftDrawerMenu.tsx

```diff
@@ -65,70 +66,70 @@ export default function LeftDrawerMenu({
   }
 
   return (
-    <div>
-      <Drawer
-        variant='permanent'
-        onMouseOver={handleDrawerOpen}
-        onMouseLeave={handleDrawerClose}
-        className={classNames(classes.drawer, {
+    <Drawer
+      data-test-id='left-drawer-menu'
+      variant='permanent'
+      onMouseOver={handleDrawerOpen}
+      onMouseLeave={handleDrawerClose}
+      PaperProps={{
+        sx: {
+          position: 'absolute',
+          background: altinnTheme.altinnPalette.primary.greyLight,
+          top: 111,
+          height: `calc(100vh - 110px)`,
+          overflow: 'hidden',
+        },
```

Comment:
Could these be set on the drawer classes?

Reply:
I guess... with the use of the `!important`-keyword in css that could be. Atm this is the way MUI suggest that we override styles on elements, at least in the documentation.